onnxruntime/core/providers/xnnpack/xnnpack_init.h

```diff
@@ -46,6 +46,19 @@ namespace xnnpack {
 #define XNN_ALLOCATION_ALIGNMENT 16
 #endif
 
+#if (!defined(_MSC_VER)) || (_MSC_VER >= 1930)
+#if defined(_M_ARM64) || defined(__aarch64__) || defined(_M_ARM64EC)
+#if !defined(__APPLE__)
+// Had to temporary disable fp16 under APPLE ARM64, as compiling
+// the source files require a hardware specific compilation flag.
+// When building an universial binary for APPLE, this flag would
+// cause trouble for x64 target.
```

Comment:
nit: we have a line length limit of 120 so these comments could use fewer lines.

Reply:
I checked those lines length are smaller than 70.